src/main.cpp

```diff
@@ -159,7 +159,8 @@ void usage(void)
        "\n"
        " -l, --localhost <hostname> Specify the local hostname or IP address\n"
        " -c, --diameter-conf <file> File name for Diameter configuration\n"
-       "     --dns-server <IP>      DNS server to use to resolve addresses\n"
+       "     --dns-server <server>[,<server2>,<server3>]\n"
```

Comment:
Can you change this to servers not server (and everywhere else that still uses dns_server?